config/routes.rb

```diff
@@ -5,7 +5,11 @@
   ActiveAdmin.routes(self)
   devise_for :users
 
-  resources :articles
+  resources :articles do
+    member do
+      put "like", to: "articles#like"
```

Comment:
us single quotes (') instead of double ("). We only use double quotes for strings that have variables that needs to be evaluated. E.g. `"Hello, #{current_user.first_name}"`